src/components/interaction-designer/block-editors/FormatStringEditor.vue

```diff
@@ -1,18 +1,23 @@
 <template>
-  <div class="block-format-string">
-    <text-editor v-model="formatString"
-        :label="'flow-builder.format-string' | trans"
-        :placeholder="'flow-builder.enter-format-string' | trans"/>
-  </div>
+  <validation-message :message-key="`block/${block.uuid}/.config.format_string`" #input-control="{ isInvalid }">
+    <div class="block-format-string">
+      <text-editor v-model="formatString"
+          :label="'flow-builder.format-string' | trans"
+          :placeholder="'flow-builder.enter-format-string' | trans"
+          :validationState="isInvalid" />
```

Comment:
@poojahpatil90 , I think we should have the same approach as https://bootstrap-vue.org/docs/components/form-input#contextual-states, because in some component we do use `bootstrap-vue`. The `validationState` prop is false for invalid, true for valid. So this line would be `:validationState="!isInvalid"` and we should update the `validationState` like this https://github.com/FLOIP/flow-builder/pull/69/files#diff-dd8be5f9484a7f9a30bd08dc9d48608cbd1a69b213ba09824f26682a6a42b15eR6 into `:class="{ 'is-invalid': !validationState }"`

What do you think ?

Reply:
Done!